models/members.js

```diff
@@ -37,6 +37,29 @@ const fetchMembers = async () => {
   }
 }
 
+/**
+ * changes the role of a new user to member
+ * @param userData { Object }: User data object to be modified
+ * @return {Promise<userModel|object>}
+ */
+
+const moveToMembers = async (userData) => {
+  try {
+    const userDoc = await userModel.doc(userData.id).get()
+    const user = userDoc.data()
+    const roles = user.roles ? { ...user.roles, member: true } : { member: true }
+    await userModel.doc(userData.id).set({
```

Comment:
Can we make use of `update()` method?
https://firebase.google.com/docs/firestore/manage-data/add-data#update-data

So we don't need the whole userData, only `id` would be required as function parameter